app/src/main/java/com/hover/stax/views/StaxDialog.java

```diff
@@ -40,9 +41,17 @@ public StaxDialog setDialogTitle(int title) {
         return this;
     }
 
+    public StaxDialog setCustomView(int viewRes) {
```

Comment:
Here, we can update the constructor to take a layout as a parameter and have one central point of inflating the layout.

Reply:
Thanks for this, it's updated in the recent push.